fusestore/store/src/meta_service/meta.rs

```diff
@@ -62,6 +64,11 @@ pub struct Meta {
 
     /// table id to table mapping
     pub tables: BTreeMap<u64, Table>,
+
+    /// A kv store of all other unclassified information.
+    /// The value is tuple of a monotonic sequence number and userdata value in string.
+    /// The sequence number is guaranteed to increment(by some value greater than 0) everytime the record changes.
+    pub unclassified: BTreeMap<String, (u64, String)>,
```

Comment:
How about changing `String` the `[u8]` or something else that does not require utf8 encoding?

Reply:
nice